css/properties/overflow.json

```diff
@@ -93,17 +93,22 @@
               "opera_android": {
                 "version_added": false
               },
-              "safari": {
-                "version_added": "preview",
-                "flags": [
-                  {
-                    "type": "preference",
-                    "name": "CSS overflow: clip support"
-                  }
-                ]
-              },
+              "safari": [
+                {
+                  "version_added": "16"
+                },
+                {
+                  "version_added": "preview",
```

Comment:
shouldn't this be replaced entirely instead ? I doubt Safari TP still requires a preference if Safari 16 shipped it.

Reply:
Uh, yes. Fixed! Thanks!